dashboard/lib/projects_list.rb

```diff
@@ -41,6 +41,28 @@ def fetch_personal_projects(user_id)
       personal_projects_list
     end
 
+    # Look up every project associated with the provided user_id, and project state, excluding those that are hidden.
+    # Return a set of metadata which can be used to display a table of personal projects in the admin UI.
+    # @param user_id
+    # @param state [String] Defaults to 'active'
+    # @return [Array<Hash>] An array with each entry representing a project.
+    def fetch_personal_projects_for_admin(user_id, state = 'active')
```

Comment:
If it's not too much trouble, I think I prefer that we just always pass state in (instead of defaulting to 'active' here).

Reply:
That makes more sense here, thanks!